libnavigator/src/main/java/com/mapbox/navigation/navigator/TripStatus.kt

```diff
@@ -3,6 +3,16 @@ package com.mapbox.navigation.navigator
 import android.location.Location
 import com.mapbox.navigation.base.trip.model.RouteProgress
 
+/**
+ * State of trip at particular timestamp
+ *
+ * @param enhancedLocation user's location
+ * @param keyPoints list of predicted locations. Might be empty
+ * @param routeProgress [RouteProgress] is progress information
+ * @param offRoute *true* if user is off-route, *false* otherwise
+ *
+ * @see [MapboxNativeNavigator.getStatus]
+ */
```

Comment:
```suggestion
 * State of a trip at a particular timestamp.
 *
 * @param enhancedLocation the user's location
 * @param keyPoints list of predicted locations. Might be empty.
 * @param routeProgress [RouteProgress] is progress information
 * @param offRoute *true* if user is off-route, *false* otherwise
 *
 * @see [MapboxNativeNavigator.getStatus]
 */
```